src/channels/domain/channel.ts

```diff
@@ -0,0 +1,21 @@
+import { ChannelType } from '../../channel-types/domain/channel-type';
+
+export class Channel {
+  id: number | string;
+
+  owner: number;
```

Comment:
In the application domain every channel must have list of users `User[]` as members.